.github/workflows/android-build.yml

```diff
@@ -52,7 +64,7 @@ jobs:
 
       - name: Create Build
         run: |
-          cd android/scripts
+          cd ${{ inputs.ANDROID_SERVICE_LOCATION }}/scripts
```

Comment:
Please remove `/scripts` as well.. 

As the ANDRIOD service location will contain the complete location